tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -1556,6 +1556,12 @@ public function testBug13310(): void
 		$this->assertNoErrors($errors);
 	}
 
+	public function testBug13714(): void
+	{
+		$errors = $this->runAnalyse(__DIR__ . '/data/bug-13714.php');
+		$this->assertCount(19, $errors);
```

Comment:
What are the 19 errors? Can you produce code that makes PHPStan crash but once fixed, it produces no errors?